src/librustdoc/html/sources.rs

```diff
@@ -16,23 +16,25 @@ use std::ffi::OsStr;
 use std::fs;
 use std::path::{Component, Path, PathBuf};
 
-crate fn render(cx: &mut Context<'_>, krate: clean::Crate) -> Result<clean::Crate, Error> {
```

Comment:
:+1: That's way cleaner. I like this change; the other comments really are just nitpicking, and the PR could easily be merged as-is.